### PR TITLE
feat: add CountEmbedder

### PR DESCRIPTION
**Pull request by simonusher**

### CODE REVIEW

File: srai/embedders/count_embedder.py [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+        the feature name (column) and value (row) e.g. amenity_fuel or type_0.
+        The rows will hold numbers of this type of feature in each region.
+
+        Args:
+            regions_gdf (gpd.GeoDataFrame): Region indexes and geometries.
+            features_gdf (gpd.GeoDataFrame): Feature indexes, geometries and feature values.
+            joint_gdf (gpd.GeoDataFrame): Joiner result with region-feature multi-index.
+
+        Returns:
+            gpd.GeoDataFrame: Embedding and geometry for each region in regions_gdf.
```
Comment (RaczeQ) [inline srai/embedders/count_embedder.py:55]: I'm not sure if we need to return geometries after embedding. Maybe we can just return `pd.DataFrame` with `region_id` index and features?

File: srai/embedders/count_embedder.py [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+            region_embeddings (pd.DataFrame): Counted frequencies of each feature value.
+
+        Returns:
+            pd.DataFrame: region_embeddings either unchanged
+                or with expected columns only.
+
+        """
+        if self.expected_output_features is not None:
+            return self._filter_to_expected_features(region_embeddings)
+        return region_embeddings
```
Comment (RaczeQ) [inline srai/embedders/count_embedder.py:81]: I don't think we have to extract those 3 lines into a separate function. It can be moved back to `embed` IMO.

**Review by RaczeQ — CHANGES_REQUESTED** [review]
Overall good job! Waiting for some tests and an example notebook 😉

File: srai/embedders/count_embedder.py (hunk 1 above)
Comment (simonusher) [reply srai/embedders/count_embedder.py]: Good idea, that's what I ended up doing.

**Review by Calychas — APPROVED** [review]

**Review by piotrgramacki — DISMISSED** [review]

**Review by RaczeQ — DISMISSED** [review]

**Review by piotrgramacki — APPROVED** [review]

**Pull request merged**